pkg/cafs/cafs.go

```diff
@@ -17,8 +17,9 @@ import (
 )
 
 const (
-	DefaultLeafSize = 2 * 1024 * 1024
-	MaxLeafSize     = 5 * 1024 * 1024
+	DefaultLeafSize     = 2 * 1024 * 1024
+	MaxLeafSize         = 5 * 1024 * 1024
+	Deduplication_blake = "blake"
```

Comment:
We should not snake-case go identifiers
```suggestion
	DeduplicationBlake = "blake"
```